### PR TITLE
Add label smooth to functional package

### PR DESCRIPTION
**Pull request by wanghaoshuang**
### PR types

Others

### PR changes

APIs

### Describe

Add label smooth to functional package

### CODE REVIEW

File: python/paddle/nn/functional/common.py [hunk 1]
```diff
@@ -1482,3 +1482,83 @@ def linear(x, weight, bias=None, name=None):
         else:
             res = tmp
         return res
+
+
+def label_smooth(label, prior_dist=None, epsilon=0.1, name=None):
```
Comment (baiyfbupt) [inline python/paddle/nn/functional/common.py:1486]: remove line23 `from ...fluid.layers import label_smooth  #DEFINE_ALIAS`

**Review by baiyfbupt — APPROVED** [review]
LGTM, except the minor issue

File: python/paddle/nn/functional/common.py (hunk 1 above)
Comment (wanghaoshuang) [reply python/paddle/nn/functional/common.py:1486]: Fixed

**Review by willthefrog — APPROVED** [review]
LGTM

**Pull request merged**